src/contracts/middleware/BLSOperatorStateRetriever.sol

```diff
@@ -8,11 +8,13 @@ import "../interfaces/IBLSPubkeyRegistry.sol";
 import "../interfaces/IIndexRegistry.sol";
 import "../interfaces/IBLSRegistryCoordinatorWithIndices.sol";
 
+import "forge-std/Test.sol";
+
 /**
  * @title BLSOperatorStateRetriever with view functions that allow to retrieve the state of an AVSs registry system.
  * @author Layr Labs Inc.
  */
-contract BLSOperatorStateRetriever {
+contract BLSOperatorStateRetriever is Test {
```

Comment:
inheriting from test + making a bunch of functions non-view (presumably want to revert these changes before merging)